src/Beberlei/Metrics/Collector/Graphite.php

```diff
@@ -101,7 +103,12 @@ public function flush()
         $this->data = array();
     }
 
-    public function push($stat, $value, $time = null)
+    /**
+     * @param string $stat
+     * @param string $value
+     * @param int|null $time
+     */
+    private function pushStat($stat, $value, $time = null)
```

Comment:
> I believed one change per commit would be enough but this PR seems to have WAY too many changes to be processed at once

One change per commit make the history easier to read, but it does not change the fact that the merge has to accept all or nothing

> However, I believe the new interface and the PHPDoc fix should be part of the same PR, don't you think?

well, as you can see, there are still some discussion about the new interface. Splitting the phpdoc fix into its own PR would allow it to be merged faster, and leaving the PR about the new interface focused on its goal.